documentation/src/main/markdown/packagemgrs/python.md

```diff
@@ -116,3 +117,27 @@ The Poetry detector parses poetry.lock for dependency information. If the detect
 The Poetry detector extracts the project's name and version from the pyproject.toml file.  If it does not find a pyproject.toml file, it will defer to values derived by git, from the project's directory, or defaults.
 
 When the `--detect.poetry.dependency.groups.excluded` property is specified, presence of both poetry.lock and pyproject.toml files is required for this detector to run successfully.
+
+## UV detector
+
+UV detector will run on your project if a pyproject.toml file containing section `[tool.uv] managed = true` is found. The UV detector extracts the project's name and version from the pyproject.toml file. If it does not find that in a pyproject.toml file, it will defer to default values.
+
+UV has two detectors:
+
+### UV CLI detector
+
+UV CLI will run if an uv executable is found along with pyproject.toml file. It will run uv tree commands to find dependencies for the project. 
+
+### UV Lockfile detector
+
+UV Lock detector will run if uv executable is not found and either an uv.lock or requirements.txt is found along with in the source directory of the project.
+
+UV Lock detector will parse uv.lock, requirements.txt or both to find dependencies for the project.
```

Comment:
UV Lock detector will parse uv.lock, requirements.txt, or both to find project dependencies.